.mypy.ini

```diff
@@ -132,3 +132,7 @@ ignore_missing_imports = True
 
 [mypy-ert.callbacks]
 ignore_errors = True
+
+[mypy-ert.config.*]
+ignore_missing_imports = True
+ignore_errors = True
```

Comment:
do we have an issue to fix the typing things mypy complains about at a later time? or is it just considered part of continuous efforts / spring cleaning or similar?

Reply:
Its an ongoing effort